exercises/simple-linked-list/example.js

```diff
@@ -0,0 +1,72 @@
+function ListElement (value, next) {
+  if (!(this instanceof ListElement))
```

Comment:
Roger that!

Also, where exactly did we set our coding standards? I remember we had a discussion about it but I can't remember if we decided to adopt a specific one.